californium-core/src/test/java/org/eclipse/californium/core/test/RequestPayloadTest.java

```diff
@@ -0,0 +1,336 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Institute for Pervasive Computing, ETH Zurich and others.
+ * 
```

Comment:
I guess, you don't work for "Institute for Pervasive Computing, ETH Zurich"?
Please replace it either with your name, or, if this is the result of your work for a company, that company name.

Reply:
I have changed the header.